src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADDBOOK = 'bookstore/src/redux/books/ADD_BOOK';
+const REMOVEBOOK = 'bookstore/src/redux/books/REMOVE_BOOK';
+
```

Comment:
- Please, kindly restructure this to follow the project requirement and conform to using the right duck's pattern below:
``` 
const ADDBOOK = 'bookstore/books/ADD_BOOK';
const REMOVEBOOK = 'bookstore/src/books/REMOVE_BOOK';
```

Reply:
@zieeco, I think I misunderstood the meaning of this. But I will change as you are indicating to me.
To be honest I am still a little confused.

Reply:
Hi @HansZizold you can refer to [this article](https://medium.com/@matthew.holman/what-is-redux-ducks-46bcb1ad04b7) for better understanding of the ducks pattern or [this repo demostrating it](https://github.com/erikras/ducks-modular-redux).